### PR TITLE
docs: include Foundry project example in `tools`

### LINKED ISSUE #190: Include Foundry project example to work with a custom Sourcify instance
Similar to https://github.com/hashgraph/hedera-sourcify/issues/175, the [tools/README](https://github.com/hashgraph/hedera-sourcify/blob/9810d773fca1d69942617356ac88d1c668b5c6e7/tools/README.md) docs include instructions to verify a contract in a custom Sourcify instance using a Foundry project. Include a Foundry project example to showcase this use case.

### PR DESCRIPTION
**Pull request by acuarica**
**Description**:

This PR includes a Foundry project example to verify contracts. Useful to make sure contract verification with Foundry works as expected.

**Related issue(s)**:

Fixes #190.

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: tools/foundry/script/Counter.s.sol [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: UNLICENSED
```
Comment (quiet-node) [inline tools/foundry/script/Counter.s.sol:1]: Shouldn't the license be Apache-2.0?

File: tools/foundry/src/Counter.sol [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: UNLICENSED
```
Comment (quiet-node) [inline tools/foundry/src/Counter.sol:1]: Shouldn't the license be Apache-2.0?

File: tools/foundry/test/Counter.t.sol [hunk 3]
```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: UNLICENSED
```
Comment (quiet-node) [inline tools/foundry/test/Counter.t.sol:1]: Shouldn't the license be Apache-2.0?

**Review by quiet-node — APPROVED** [review]
LG but some Qs

File: tools/foundry/script/Counter.s.sol (hunk 1 above)
Comment (acuarica) [reply tools/foundry/script/Counter.s.sol:1]: I wanted to keep Foundry's default

**Pull request merged**